cloudformation/service/eu-west-2/rds.yml

```diff
@@ -62,11 +72,20 @@ Resources:
         aurora_load_from_s3_role:
           Fn::ImportValue: !Sub ${Stage}:RdsAuroraS3RoleArn
 
+  DatabaseClusterParameterGroup5dot7:
+    Type: AWS::RDS::DBClusterParameterGroup
+    Properties:
+      Description: Parameter Group for master DB
+      Family: aurora-mysql5.7
+      Parameters:
+        aurora_load_from_s3_role:
+          Fn::ImportValue: !Sub ${Stage}:RdsAuroraS3RoleArn
+
   DatabaseCluster:
     Type: AWS::RDS::DBCluster
     Properties:
-      Engine: aurora
-      EngineVersion: 5.6.10a
+      Engine: !If [IsTest, aurora-mysql, aurora]
```

Comment:
I upgraded test to 5.7 but haven't moved the other environments yet, there's a ticket in the backlog but will need a bit of planning since it might result in a bit of downtime

Reply:
Cool, we'll schedule this in as discussed